src/createInstance.ts

```diff
@@ -228,6 +228,8 @@ export function createInstance(
               this.$.setupState[k] = v
               // eslint-disable-next-line no-empty
             } catch (e) {}
+            //@ts-ignore Setting an unknown global variable for the component instance
+            globalThis[k] = v
```

Comment:
Did this work in Test Utils v1? I actually am surprised this doesn't "just work".